include/Network/HTTP/BadGatewayException.php

```diff
@@ -9,6 +9,7 @@
 /**
  * @brief Bad Gateway (502) http exception
  */
-class BadGatewayException extends HTTPException {
+class BadGatewayException extends HTTPException 
+{
```

Comment:
I think it makes sense since the class is a special control structure, pretty different from functions, loops or conditionals.
Additionally, a class declaration can be quite long and elaborate with class and interface inheritance, so the opening brace could not always be included in the standard 80 columns display console can have.
Finally, I'd like us to steer as little away as we can from PSR-2 since maintaining a large Friendica-specific set of coding standards is neither fun nor smart in my opinion.